lib/middlewares.ts

```diff
@@ -3,7 +3,14 @@ import jwt, { JwtPayload } from "jsonwebtoken";
 import jwks , { CertSigningKey, RsaSigningKey } from "jwks-rsa";
 
 import * as Constants from "../constants";
+import { createClient } from "./graphql";
+import { getAuthToken } from "./environment";
 import MiddlewareError from "../utils/MiddlewareError";
+import { FetchAppDetailsDocument, FetchAppDetailsQuery } from "../generated/graphql";
+
+interface DashboardTokenPayload extends JwtPayload {
+  app: string;
```

Comment:
I think `appId` would be clearer

Reply:
I don't have any opinion here, but just be aware, that's a shape of JWT token - if we want to change it it will be breaking change in Saleor.

Reply:
Oh, okay 👍 Let's keep it as it is then